multiqc_cmgg/modules/msi_sensor_pro/msi_sensor_pro.py

```
@@ -58,27 +69,196 @@ def __init__(self):
+                    value = sample_data.get(col_key, '')
+                    # Apply formatting and conditional colors
+                    if col_key == 'perc':
+                        try:
+                            val = float(value)
+                            css_class = ''
+                            if val >= self.msi_high_threshold:
+                                css_class = 'class="bg-danger"'  # Red for high percentage
+                            else:
+                                css_class = 'class="bg-light-success"'  # Light green for normal
+                            html_parts.append(f'<td {css_class}>{value:.2f}%</td>')
+                        except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)